motion_server_node:
  ros__parameters:
    move_group_name: "ur_robot"
    end_effector_name: "ur10e_tool0"
    base_link_name: "base_link"
    joint_names:
      - 'ur10e_shoulder_pan_joint'
      - 'ur10e_shoulder_lift_joint'
      - 'ur10e_elbow_joint'
      - 'ur10e_wrist_1_joint'
      - 'ur10e_wrist_2_joint'
      - 'ur10e_wrist_3_joint'

    cartesian_max_step: 0.0025
    cartesian_fraction_threshold: 0.0
    cartesian_jump_threshold: 0.0
    cartesian_avoid_collisions: True
    max_velocity: 0.5
    max_acceleration: 0.5